### ISSUE TITLE
"no proxy for" for  automatic mode

### ISSUE THREAD

**Issue by hesss5** [rNeomy/proxy-switcher#15]
is there anyway add "no proxy for" for automatic mode?
like manual mode?

thanks

**Comment by rNeomy** [comment]
I don't think it is possible. I couldn't find a setting for this. However, you might be able to use PAC script for this purpose.

**Comment by hesss5** [comment]
thanks for replay,to complicated with this way.hope you find easy way :)
i use this
https://addons.mozilla.org/en-US/firefox/addon/windscribe/reviews/?src=api
how to send this both url in automatic mode with PAC script for above addon?
google.com
gstatic.com
